doc/how_to_start.rst

```diff
@@ -0,0 +1,72 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _how_to_start:
+
+
+##########################################
+How to run your model
+##########################################
+
+The installation procedure installs PyPSA-Earth model with all the software dependencies needed to build and run it. To properly model any region of the Earth, PyPSA-Earth needs to download and fetch different datasets. This section explains how to perform this data management.
+
+Tutorial run
+------------------------------
+
+The whole global data kit is very heavy (e.g. for Africa it is >40Gb) and involves a large number of files. To facilitate model testing, a lightweight data starter kit was developed. You can use it by setting in the configuration file `config.yaml` (placed in the project folder `pypsa-earth`) the following flags:
+
+.. code:: yaml
+
+    tutorial: true
+    ...
+    retrieve_databundle: true
+
+After doing so, you just need run the modeling workflow with the following command:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks
+
+.. TODO Explain settings of the tutorial case
+
+This command will trigger loading of the whole dataset needed to build the model for a tutorial case that is simulation of power systems in Nigeria and Benin. Note please that data loading and model building will take a while (about 20..50 minutes).
+
+Working run
+------------------------------
+
+After playing with the tutorial model, it's important to clean-up data in your model folder to avoid data conflicts. You may use the `clean` rule for making so:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 clean
+
+Generally, it's a good idea to repeat the cleaning procedure every time when the underlying data are changed.
+
+It's recommended to set `retrieve_databundle: true` when building the model first time to download the common data files needed. The load will start automatically when running the model with:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks
+
+Please, note that around **20 Gb zipped files will be downloaded**. It's worth to make sure you have a stable connection, time and around 50 Gb available in your system. If no errors show up, then you can proceed. It's advisable to set `retrieve_databundle: false` after the first model run when all the needed data will be successfully extracted to avoid data loss.
+
+Snakemake
```

Comment:
Maybe change to question?

Snakemake -> `How to snakemake rules in PyPSA-Earth?  # (AIM: Show that people can execute different rules) (see PyPSA-Eur)

Currently missing is also this part:
Config changes -> `How to customize PyPSA-Earth?  # (AIM: Introduction to config.yaml & show that people can change values ... there is a need to clean as you wrote above before executing after the change) --- Similar stuff is written in PyPSA-Eur

Reply:
Again, my feeling is that it relates rather to a detailed tutorial

Reply:
What is the detailed tutorial? You mean the hackathon? 
We just need some hands-on stuff so people get introduced to the capabilities
Anas is working only on the analysis of the resulting network atm